frida_tools/application.py

```diff
@@ -18,6 +19,7 @@
 import colorama
 from colorama import Cursor, Fore, Style
 import frida
+import shlex
```

Comment:
This should be moved up to the first group of imports. The first group is for imports from the Python standard library, the second is for external imports.

Reply:
Done.